pkg/update/update.go

```diff
@@ -459,3 +471,93 @@ func updateNoIP(httpClient *http.Client, hostname, username, password, ip string
 	}
 	return "", fmt.Errorf("unknown response: %s", s)
 }
+
+func updateDnsPod(httpClient *http.Client, domain, host, token, ip string) (newIP string, err error) {
+	postValues := url.Values{}
+	postValues.Set("login_token", token)
+	postValues.Set("format", "json")
+	postValues.Set("domain", domain)
+	postValues.Set("length", "200")
+	postValues.Set("sub_domain", host)
+	postValues.Set("record_type", "A")
+	req, err := http.NewRequest(http.MethodPost, "https://dnsapi.cn/Record.List", bytes.NewBufferString(postValues.Encode()))
+	if err != nil {
+		return "", err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	resp, err := httpClient.Do(req)
+	if err != nil {
+		return "", err
+	}
+	respBody, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
+	if err != nil {
+		return "", err
+	}
+	type RecordResp struct {
```

Comment:
Please use anonymous structs for a simpler code, similarly to [these 5-10 lines](https://github.com/qdm12/ddns-updater/blob/master/pkg/update/update.go#L266)